tools/pip-requires-test

```diff
@@ -23,3 +23,4 @@ pytz==2017.3                # World timezone definitions, modern and historical
 Babel==2.5.1                # Internationalization utilities - Dependency of sphinx
 subprocess32==3.2.7; python_version <= '3.0' # A backport of the subprocess module from Python 3.2/3.3 for use on 2.x.
 pycodestyle==2.3.1          # New package replacing pep8
+fts3-rest-API==3.7.1        # FTS rest API
```

Comment:
This should be moved to pip-requires as well, as it is not only used for testing.

Reply:
yes. 
side comment It brings M2Crypto and pycurl. Not the most pythonic dependency..

Reply:
mmm, it is already in pip-requires, isn't it (*)?
Regarding the dependencies I could agree, but we also did agree to push for having these bindings from FTS people. So I don't know how you prefer to proceed at this point.

(*)https://github.com/rucio/rucio/pull/1178/files#diff-3f6c71b838e7a93232a1c4b91bb669f7R44

Reply:
Ooops 🙄 Sorry 😄 
In that case you can remove it from pip-requires-test as pip-requires will be installed for sure anyway.

Reply:
np ^^ Removed from test then.